Musicplayer.css

```diff
@@ -22,62 +25,68 @@ body{
 
 nav{
 	width:100%;
-	background:white;
+	background-color: #0C59CF;
 	overflow:auto;
+	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
 
 li{
-	float:left;
+	float:right;
```

Comment:
21/9: I change the link in the nav to the right because I think the nav bar in the right side look a bit empty. So I will make the logo to be in the left side and the link to be in the right side so the nav will look more blance.